packages/react-select/src/Select.js

```diff
@@ -516,7 +531,7 @@ export default class Select extends Component<Props, State> {
     this.setState({
       menuOptions,
       focusedValue: null,
-      focusedOption: menuOptions.focusable[openAtIndex],
+      focusedOption: focusDefaultOption ? menuOptions.focusable[openAtIndex] : null,
```

Comment:
This might have the interesting effect of calling `this.announceAriaLiveContext({ event: 'menu' });` twice if the user uses the keyboard to navigate the Select component. The first time will be in the callback of this call to `setState` (see a few lines below this one). The second time will be in `focusOption` where it calls `this.announceAriaLiveContext({ event: 'menu' });` again [if there is no selected option](https://github.com/JedWatson/react-select/blob/master/packages/react-select/src/Select.js#L585-L589).

I believe we can remove the second case because there is not a time where we call `focusOption` when the menu is closed (https://github.com/JedWatson/react-select/pull/3427 removed the last case of this situation) and should do that as part of this PR in order to avoid the above scenario.